src/pages/[platform]/tools/libraries/configure-categories/index.mdx

```diff
@@ -26,10 +19,207 @@ export function getStaticProps(context) {
   };
 }
 
-import js0 from '/src/fragments/lib-v1/client-configuration/js/js-configuration.mdx'; 
+<InlineFilter filters={['javascript', 'react-native', 'angular', 'nextjs', 'vue','react']}>
 
-<Fragments fragments={{ javascript: js0, angular: js0, nextjs: js0, react: js0, vue: js0 }} />
+When using the Amplify CLI, the **aws-exports.js** file gets created and updated automatically for you based upon the resources you have added and configured.
 
-import reactnative0 from '/src/fragments/lib-v1/client-configuration/js/js-configuration.mdx';
+If you are not using the Amplify CLI or need to override these settings, this documentation shows the available configuration properties for each category.
 
-<Fragments fragments={{ 'react-native': reactnative0 }} />
\ No newline at end of file
+## General configuration
+
+The `Amplify.configure()` configuration can generally be included in your project _entrypoint_ file (i.e. _/src/main.js_). It takes in two parameters - `ResourcesConfig` and `LibraryOptions`.
+
+```ts
+Amplify.configure(
+  {}: ResourcesConfig;
+  {}?: LibraryOptions;
+);
+```
+
+## Authentication (Amazon Cognito)
+
+```js
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      userPoolClientId: string;
+      userPoolId: string;
+      userPoolEndpoint?: string;
+      signUpVerificationMethod?: 'code' | 'link';
+      loginWith?: {
+        oauth?: OAuthConfig;
+        username?: boolean;
+        email?: boolean;
+        phone?: boolean;
+      };
+    }
+  }
+});
```

Comment:
This is defining the types but not an example.

Reply:
Good point, will update to show example values